### PR TITLE
VO RDF's fixed

### PR DESCRIPTION
**Pull request by ajstanley**

### CODE REVIEW

File: includes/ingest_versionable_object.form.inc [hunk 1]
```diff
@@ -624,6 +624,8 @@ function islandora_critical_edition_advanced_ingest_versionable_object_tei_rdf_p
   foreach ($dsids as $dsid) {
     if (isset($page[$dsid])) {
       $datastream = $edition->constructDatastream($dsid, 'M');
+      $unique_id = uniqid();
+      $file = file_save_data('', "temporary://$unique_id_datastream_$DSID");
       $file = file_save_data('', 'temporary://datastream');
```
Comment (nigelgbanks) [inline includes/ingest_versionable_object.form.inc]: Shouldn't this one be removed as well?

File: includes/calliope.inc [hunk 2]
```diff
@@ -388,7 +388,7 @@ function islandora_calliope_write_mvd($newmvd) {
   $newobj = array('_revisions' => array($newmvd), 'metadata' => $newmvd);
   $db = islandora_calliope_create_mongo_db();
   $coll = $db->selectCollection('mvds');
-  $inserted = $coll->insert($newobj, array('safe' => TRUE));
+  $inserted = $coll->insert($newobj, array('w' => 1));
```
Comment (nigelgbanks) [inline includes/calliope.inc:391]: What consequences does removing  `'safe'` have? What does does the the `'w'` value do differently? Could you add a comment that links to the documentation for these properties if it exists?

Comment (ajstanley) [inline includes/calliope.inc:391]: these are the properties associated with the new MongoClient class, which replace the previous Mongo class.

Comment (MorganDawe) [inline includes/calliope.inc:391]: A good reference to why this had to change can be found here: http://derickrethans.nl/mongoclient.html

**Pull request merged**